<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система распознавания лиц</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <item>
       <widget class="QLabel" name="video_label">
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>360</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: black; border: 2px solid gray; color: white;</string>
        </property>
        <property name="text">
         <string>Камера неактивна</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <item>
       <widget class="QPushButton" name="btnRegister">
        <property name="text">
         <string>Зарегистрировать пользователя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRecognize">
        <property name="text">
         <string>Запуск распознавания</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnShowDB">
        <property name="text">
         <string>Просмотр базы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnThresholds">
        <property name="text">
         <string>Изменить пороги</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
